content/en/docs/reference/controllers/replicationcontroller.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
This document should probably be merged into the replicaset.md document and/or be deleted since this is a deprecated thing. Not in this PR, of course... just a thought for a later PR.

Reply:
There is a replication controller, and it's not the same bit of code as the ReplicaSet controller.

I would hyperlink the two but not merge them.

Reply:
ack, hyperlink++

Reply:
Hyperlinked